scanpy/external/__init__.py

```diff
@@ -34,6 +34,11 @@
    pp.harmony_integrate
    pp.mnn_correct
 
+Multiplet detection
+~~~~~~~~~~~~~~~~~~~
+
+   pp.scrub_doublet
```

Comment:
```suggestion
   pp.scrublet
   pp.scrublet_simulate_doublets
   pl.scrublet_score_distribution
```

Looks like this didn't get updated